packages/ibm-products/src/components/DataSpreadsheet/DataSpreadsheet.tsx

```diff
@@ -299,7 +299,11 @@ export let DataSpreadsheet = React.forwardRef(
                   activeCellCoordinates?.column
                 ]
               : null;
-          setActiveCellContent(activeCellFullData?.render('Cell') ?? null);
+          if (activeCellFullData) {
```

Comment:
@matthewgallo Based on the videos here in the issue https://github.com/carbon-design-system/ibm-products/issues/5526 do you know if this is just a typescript issue? Because it looks like the null check originally does not work as intended when we import it through our packages on our code. I'm not that familiar with TypeScript outside the basics, not sure on the syntax specific differences there are. Can you let me know if you have any understanding? Thanks!

Would like to get this PR merged in as soon as possible if this is a true fix for what I am seeing on our local env. This is actually a regression in 2.40 for us though on storybook it wasn't able to be replicated, like I mentioned in the issue it looks like due to how it gets imported to the packages, the null checks get constructed weirdly. Thanks!

Reply:
Sure, I'll take a look.

Reply:
I'll make a video and send it over on slack after lunch. Thanks!